docs/template_plugin/tests/functional/subgraph_reference/preprocess.cpp

```diff
@@ -101,23 +101,44 @@ static RefPreprocessParams scale_then_mean() {
     return res;
 }
 
+static RefPreprocessParams convert_only() {
+    RefPreprocessParams res("convert_only");
+    res.function = []() {
+        auto f = create_simple_function(element::f32, Shape{1, 1, 2, 2});
+        f = PrePostProcessor().input(InputInfo()
+                .tensor(InputTensorInfo().set_element_type(element::i16))
+                .preprocess(PreProcessSteps()
+                .convert_element_type(element::f32)
+                .scale(3.f)
+                .convert_element_type(element::u8)
+                .convert_element_type(element::f32)))
+                        .build(f);
+        return f;
+    };
+    res.inputs.emplace_back(Shape{1, 1, 2, 2}, element::i16, std::vector<int16_t>{2, 3, 4, 5});
+    res.expected.emplace_back(Shape{1, 1, 2, 2}, element::f32, std::vector<float>{0., 1., 1., 1.});
+    return res;
+}
+
 static RefPreprocessParams convert_element_type_and_scale() {
     RefPreprocessParams res("convert_element_type_and_scale");
     res.function = []() {
-        auto f = create_simple_function(element::i8, Shape{1, 3, 2, 2});
+        auto f = create_simple_function(element::u8, Shape{1, 3, 2, 2});
```

Comment:
Why did you change `i8` to `u8`?

Reply:
I stated in details, that reference implementation of 'convert to int8' has different rounding rules comparing to CPU/GPU, etc. Ticket for this is 66520. So I changed int8 to uint8 in preprocessing tests so that these are not dependent on this issue